server/app/views/applicant/ProgramOverviewTemplate.html

```diff
@@ -15,51 +15,61 @@
           #{heading.programOverview(${programName})},
           ${programDescription})}"
       ></div>
+
       <div class="grid-container">
-        <a
-          th:href="${actionUrl}"
-          class="usa-button"
-          th:text="#{button.startApp}"
-        ></a>
-        <div class="usa-prose section" data-testId="steps-and-alerts">
-          <h2 th:text="#{heading.applicationSteps}"></h2>
-          <ol class="usa-process-list">
-            <li
-              th:each="applicationStep : ${applicationSteps}"
-              class="usa-process-list__item"
-            >
-              <h3
-                th:text="${applicationStep.getKey()}"
-                class="usa-process-list__heading"
-              ></h3>
-              <p th:utext="${applicationStep.getValue()}"></p>
-            </li>
-          </ol>
-          <div th:if="${showEligibilityAlert}" data-testid="eligibility-alert">
-            <div th:if="${isEligible}">
-              <div
-                th:replace="~{components/AlertFragment ::
-            alert(alertSettings=${eligibilityAlertSettings}, headingLevel='H6')}"
-              ></div>
-            </div>
-            <div th:if="${!isEligible}">
-              <div
-                th:replace="~{components/AlertFragment ::
-            alert(alertSettings=${nonEligibilityAlertSettings}, headingLevel='H6')}"
-              ></div>
-            </div>
-          </div>
-          <div th:if="${isGuest}" class="section-internal">
-            <div
-              th:replace="~{applicant/CreateAccountAlertFragment :: createAccount(${createAccountLink}, ${actionUrl}, #{link.createAccountFromOverview}, #{link.startAsGuest})}"
-            ></div>
-          </div>
-          <div th:if="${!isGuest}" class="section-internal">
+        <div class="grid-row">
+          <div class="tablet:grid-col-8">
```

Comment:
Others pages (e.g applicant upsell, form) use `"tablet:grid-col-8"` to limit the max width. We shouldn't use Tailwind classes, but couldn't find another way to do this with USDWS. `usa-prose` sets the max characters of the text, but it doesn't affect the alert. Adding `usa-prose` inside the alert wraps the text, but the alert still expands all the width.

We could set the max length.. but would need to make it reactive. I am not sure what is the best thing to do here. Any thoughts?

Reply:
The `tablet:grid-col-8` and related classes are part of [USWDS utilities](https://designsystem.digital.gov/utilities/layout-grid/) and should work. Tailwind might have something named the same, but USWDS is loaded after Tailwind so that should take precedent. Let me know if that doesn't work.

Reply:
Oh if we can use `tablet:grid-col-8` then this does wrap the content, so should be good for review :)